app/controllers/api/v1/short_form_controller.rb

```diff
@@ -6,6 +6,14 @@ def submit
     logger.debug "application_api_params: #{application_api_params}"
     short_form_validator = ShortFormValidator.new(application_api_params)
     if short_form_validator.valid?
+      # If lease information is available, submit it separately.
```

Comment:
I think this comment is not necessary, it's pretty clear what is happening, at least to me